UserSync/Controllers/AccountController.cs

```diff
@@ -107,7 +116,7 @@ public ActionResult SignOutCallback()
         /// </summary>
         private void RemoveCachedTokens()
```

Comment:
RemovedCachedTokensForApp?
It's obvious to us because it's a daemon, but what if customers use daemon + OBO in the same app?